### PR TITLE
Record calcium concentration new

### PR DESCRIPTION
**Pull request by katduecker**
Sorry to have opened a new PR, I had some merging conflicts. This one supersedes #792

We decided that the absolute values of the calcium concentration may not be interpretable, which is not a problem, but probably something we want to state in publications that use the feature!

### CODE REVIEW

**Comment by kmilo9999** [comment]
@katduecker you probably need to rebase this branch in order to run successfully the checklist.

**Comment by katduecker** [comment]
Thanks!

**Comment by ntolley** [comment]
Looks like this is the offending line:
image

Just to double check, does the `soma` actually have calcium channels? Or should this be changed to one of the dendrites.

**Comment by katduecker** [comment]
Yep, somehow cell_response.ca was in the wrong line. Thanks for pointing me to it :)

File: doc/whats_new.rst [hunk 1]
```diff
@@ -200,7 +200,7 @@ API
   :meth:`~hnn_core.extracellular.ExtracellularArray.plot_lfp`, by
   `Steven Brandt`_ and `Ryan Thorpe`_ in :gh:`517`.
 
-- Recorded voltages/currents from the soma, as well all sections, are enabled by
+- Recorded voltages/currents from the soma, as well as all sections, are enabled by
```
Comment (ntolley) [inline doc/whats_new.rst:203]: much appreciated :wink:

File: doc/whats_new.rst [hunk 2]
```diff
@@ -210,6 +210,11 @@ API
   :class:`~hnn_core.Network` objects, 
   by `Nick Tolley`_ and `Ryan Thorpe`_ in :gh:`619`.
 
+- Recorded calcium conncetration from the soma, as well as all sections, are enabled
+  by setting `record_ca` to `soma` or `all` in :func:`~hnn_core.simulate_dipole`.
+  Recordings are accessed through :class:`~hnn_core.CellResponse.ca`, 
+  by `Katharina Duecker`_ in :gh:`792`
```
Comment (ntolley) [inline doc/whats_new.rst:216]:
```suggestion
  by `Katharina Duecker`_ in :gh:`804`
```

File: hnn_core/cell.py [hunk 3]
```diff
@@ -783,10 +791,35 @@ def record(self, record_vsec=False, record_isec=False):
...
+            self.ca = dict.fromkeys(section_names)
+
+        if record_ca:
+            for sec_name in self.ca:
+                if hasattr(self._nrn_sections[sec_name](0.5), '_ref_cai'):
+                    self.ca[sec_name] = h.Vector()
+                    self.ca[sec_name].record(
+                        self._nrn_sections[sec_name](0.5)._ref_cai)
+
+        # calcium concentration
```
Comment (ntolley) [inline hnn_core/cell.py:810]: This section seems duplicated?

**Comment by ntolley** [comment]
Great work @katduecker!!  We should definitely put together an example showing off how the `calcium_model()` and the `jones_2009_model()` differ using this feature.

**Comment by ntolley** [comment]
Also can you update the title to have the [MRG] tag and get rid of "new"?

**Pull request merged**